packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -75,36 +72,53 @@ TIP: If your log file is part of your Salesforce DX project, you don’t need to
 
 ## Debug Your Code
 
-Set breakpoints, then replay your debug log and inspect your variables’ values.
+Set breakpoints or checkpoints, then replay your debug log and inspect your variables’ values.
 
 1. To set line breakpoints, open a `.cls` or `.trigger` file and click the column to the left of the line numbers.
+1. For more information than line breakpoints provide, add checkpoints. You can set up to five checkpoints to get heap dumps when lines of code run. All local variables, static variables, and trigger context variables have better information at checkpoints. Trigger context variables don’t exist in logs and are available only at checkpoint locations.  
+In Visual Studio Code, a checkpoint is a type of breakpoint. Set up and upload your checkpoints, then start an Apex Replay Debugger session.  
```

Comment:
I think the sequence here is a little off.
1. Set up and upload your checkpoints.
2. Turn on Apex Debug Logs for Replay Debugger (SFDX Turn on Apex Logs for Replay Debugger)
3. Run your code
4. Pull the Apex Log (SFDX: Get Apex Debug Logs...)
5. Start an Apex Replay Debugger Session